onnx/defs/parser.cc

```diff
@@ -20,6 +20,7 @@ namespace ONNX_NAMESPACE {
 
 Status ParserBase::Parse(Literal& result) {
   bool decimal_point = false;
+  bool is_float = false;
```

Comment:
What's the difference between `decimal_point` and `is_float`? Is this just a variable renaming?